### DISCUSSION TITLE
Launch Creative Origin website

### DISCUSSION THREAD

**Discussion by bafu** [creative-origin/.github#12] [General]
Jesse and I pitched Creative Origin to some people. During the conversations, we realized that a well-designed website would help communicate our concept, share news, and promote upcoming workshops.

We will discuss topics related to the website creation here, including:

- Website content
- Website design
- Sharing beta previews and collecting feedback
- Milestones and schedule

**Comment by JesseJamesRingham** [comment]
Thanks Bofu, yes it's important to have a simple, clean website for the creative partners to view updates on Creative Origins, and also for press and marketing to link back to please. The goal of the website is to have a universal home for all the conversation, news and actions happening all over the world with regards to AI and creative copyright. At present info is scattered, many companies, many teams, many policies... this public site will serve as a central space, a unified, collective voice. 

This could be too much information here, but please feel free to strip back!

**Website Outline and Design Concept for Creative Origins Alliance**
Logo Inspiration

- Similar to the C|O (Creative Origins) logo on [C/O Berlin](https://co-berlin.org/en).
- Modern, minimalist design emphasizing the initials "C|O" with a creative twist.
- Clean lines, possibly incorporating a subtle graphic element representing creativity and collaboration.

Navigation Menu

**Homepage**
- Strapline: "Fostering Creativity and Innovation Across Industries"
- Rotating image banner showcasing projects, events, and key alliance members.
- Quick links to recent news, upcoming events, and featured reports.

**News**
- Sections for updates from each C|O Alliance partner.
- Categories: Progress, Policies, Reports.
- Search and filter options for news articles and reports.

**About**
- Subsections: Mission, Ethics, Goals.
- Detailed descriptions of policy and technical solutions.
- Overview of alliance composition: creative industries, technology companies, policy leaders, legal entities, institutions, education, news media bodies.

**Events**
- Calendar view of upcoming talks, discussions, roundtables, and conferences.
- Event details: date, time, location (physical/virtual), and registration links.
- Archive of past events with summaries and recordings if available.

**Content by page**
Homepage
- High-quality, dynamic images that change to reflect different alliance activities.
- Clear, impactful strapline.
- Teasers for News, About, and Events - linking off to each area of the site
- Highlight recent news articles, key policy reports, and the next major event.
_- Featured Partners Section_
1. Logos and brief descriptions of key alliance members.
2. Clickable links to individual partner pages or external sites.

**News Page**
- Main News Feed
- Layout with headlines, images, and summaries. For 3rd party articles, reports or news just use headlines and links out to content
- Categories/tags for easy navigation.
- Filters and Search
- Filter by partner, date, or category.
- Search bar for specific reports or news items.

**About Page**
- Mission Section
- Clear statement of the alliance's mission and vision.
- Quotes from key figures in the alliance.
- Ethics and Goals Section
- Detailed descriptions of ethical standards and long-term goals.
- Infographics or visual aids to represent data and strategies.
- Alliance Composition
- Interactive infographic or directory of member industries and key partners.
- Short descriptions and links to detailed profiles of each member.

**Events Page**

- Upcoming Events Calendar
- Interactive calendar with clickable dates for event details.
- Featured upcoming events with banners or highlighted sections.
- Event Details
- Full descriptions of events, speakers, and agendas. or just link out to 3rd party event
- Registration links and options for calendar integration (e.g., Google Calendar, Outlook).

**Responsive Design**
Mobile-friendly layout ensuring accessibility on various devices.
Consistent Branding
Use of a cohesive colour scheme and typography throughout the site.
Logo presence on all pages

**Accessibility Features**
Alt text for images, keyboard navigation, and screen reader compatibility.
Consideration for colour contrast and font size to aid readability.

**Sign up and Communications to partners**
A visible line on each page saying **Join Creative Origins** and email sign up on each page.
Alliance comms updates to partners - to be through email newsletter or Slack.

Examples of website design:
https://creativecommons.org/
https://www.weforum.org/
https://www.the-aop.org/
https://www.dacs.org.uk/

**Reply by bafu** [reply]
@JesseJamesRingham I created a [raw wireframe](https://miro.com/app/board/uXjVKq2UXME=/?share_link_id=72351086448) quickly and will keep updating it. You can feel free to edit it, and let's discuss more tomorrow!

**Reply by JesseJamesRingham** [reply]
Hi Bofu, looks great! Lets discuss tomorrow on a screen share.. I like the partner groups.

**Comment by bafu** [comment]
@JesseJamesRingham, Thank you for sharing your informative thoughts. One quick question: What kind of color scheme or style do you have in mind?

**Reply by JesseJamesRingham** [reply]
Hello @bafu I would keep similiar to https://protectingcreativity.numbersprotocol.io/ but black text and white background, keep design aligned more to arts, media and creative industrys, so not web3 or tech in design - hope that helps! More examples below.

Examples of website design:
https://creativecommons.org/
https://www.weforum.org/
https://www.the-aop.org/
https://www.dacs.org.uk/

**Comment by aphxbt** [comment]
Hi @JesseJamesRingham , I’m Andy, the designer who will be working on the designs for Creative Origin. After catching up with all the comments, replies, and examples you provided (some amazing stuff), I think we’re in a pretty good place to move forward! I’ve also started working on the logo and will share it here as soon as it’s finished. After that, I’ll dive into the website design. Do you have any suggestions on how we should approach the content for the website?

**Reply by JesseJamesRingham** [reply]
Great to meet you @andyandesign.
I’m back online tomorrow afternoon UK briefly and will reply in much more detail. Essentially the public site for creative origins is a communication tool for all partners where they can share events, reports and progress in creative AI policy.. in terms of icons I spotted this site today, a good example https://futureartecosystems.org/ to have a look at?  Content will come from partners, as if they’re contributors. We’ll also feature and syndicate news from the Ai x copyright industry. If Bofu agrees first with the above. I’m texting in my mobile, so apologies for any typos!

**Reply by bafu** [reply]
@andyandesign Because @JesseJamesRingham wants to share the site with some partners next week, do you think it's possible to share the current design materials with us? Anything available is ok. Let me think about how to make impossible possible 😎. We will keep updating and improving the website in the next sprints.

**Reply by aphxbt** [reply]
Sure, I haven’t started on the website design yet because I have other tasks on hand at the moment(let's find a time to chat about the timeline for this), but here’s what I have for the logo:
> The style is clean, modern, and minimalistic, with a creative twist to the letters C and O. The way the positive and negative spaces interact to form two overlapping circles, one emerging from the other, conveys the concept of origin and creativity.

![Artboard 1-100](https://github.com/user-attachments/assets/80c72e20-8fa2-442e-be13-017c0df145ae)
![Artboard 2-100](https://github.com/user-attachments/assets/44b313ab-44ad-47fc-ad5a-c281248a61c4)
![Artboard 3-100](https://github.com/user-attachments/assets/b06169ca-6352-429c-b7e9-90d24b7c71ed)

**Comment by bafu** [comment]
@newbreedofgeek @ETeissonniere @SailorSnoW @robertu7 @nwingt @Orange-Web3 @mrrifat

We are planning to build the Creative Origin website, and all the design concepts and materials will be public here. 

Please feel free to share your feedback or thoughts. Thanks!

**Comment by aphxbt** [comment]
Hi @JesseJamesRingham , I’ve started designing the landing page, and here’s the basic structure for v1 of the website. I wanted to give you a look before we move forward. Let me know what you think!
![Creative Origin Homepage](https://github.com/user-attachments/assets/412d7602-4ccb-45a5-8f6d-030d987f0ba6)

**Reply by aphxbt** [reply]
I've started building the v1 website, please feel free to play with it: https://creative-origin.squarespace.com/
the password is: creativeorigin

**Reply by JesseJamesRingham** [reply]
Looks great! We need an email sign up too, so people can be notified of news, events and developments in the ethical use of AI please

**Reply by JesseJamesRingham** [reply]
When do you think the site will be completed please?

**Reply by JesseJamesRingham** [reply]
Suggest including:
1. **Upcoming Events** (conferences, talks), **News** (commentary in the press) and **Developments** (technical advancements, reports released and policies
2. A functionality to for partners to comment under each mention? It'll will mean a little moderation form the team - but we need this website to be a communication tool too @bafu if you agree?
3. Add **Upcoming Events** **News** and **Developments** also as menu links at the top and across the site?
4. Add a newsletter sign up to get notified occassionally by C|O
5. Not **Testimonials** but use **Partners** please as the headed - include the name of the company, face of person and their title - more emotive!

**Reply by bafu** [reply]
@andyandesign FYI: @Cheerock and @JesseJamesRingham are [collecting the website content](https://github.com/orgs/creative-origin/discussions/12#discussioncomment-10541500), and we aim to make the website online before the end of next week.

**Reply by bafu** [reply]
@JesseJamesRingham The website is online: https://creativeorigin.ai/

Let's keep updating and improving it 👍

**Comment by Cheerock** [comment]
@JesseJamesRingham  
Here is the document for us all to co-work on the copy for the website https://docs.google.com/document/d/135zbz1fWlSsN7zahqXxV6g9fRKLmO7iX8oZP_FfYKuM/edit

**Comment by JesseJamesRingham** [comment]
**A. Homepage**
- Add a visually appealing data visualisation or motion image to the homepage and Members page. Consider using free resources or generating custom visuals that represent the creative industries and AI intersection. 
<img width="1189" alt="Screenshot 2024-09-12 at 09 51 52" src="https://github.com/user-attachments/assets/d2530d0d-1433-435e-bb17-2b15f314f289">

- Examples provided in comments below.

- Content Structure:
i) Remove "GOALS & PURPOSE" and just have "PURPOSE"
ii) Display all content without cascading UX, making it immediately visible. See below.

Add icons to each point for visual appeal, more stylish/professional than emoticons I've used (!) try www.thenownproject.com:

🖼️ Copyright. Advocate for creators' rights, addressing copyright issues and art ownership.
📜 Policy. Engage with policymakers in UK, USA, and EU to influence changes in creative industries and AI.
📋 Guidelines. Develop ethical guidelines for Generative AI representing artists' interests.
🤝 Collaboration. Work with stakeholders across technology, legal, policy, and creative sectors to engage AI companies using copyrighted assets.
🎓 Education. Conduct workshops with cultural organisations on navigating generative AI.

- Partner Showcase: Add a swipeable list of partner logos on the homepage below 'News' section. Each click through to their partner page.

- Homepage intro copy, revise the copy to break it down and improve readability:
"Creative Origin is an alliance of the creative industry, artists, technology companies, legal experts, and policymakers focused on the intersection of AI and creative industries.

We track developments in Creative AI, monitor copyright regulations, promote ethical AI development, protect artists' ownership rights, and encourage transparency and innovation.

Our shared goal is to balance technological advancement with creative rights in the evolving landscape of AI-generated content."

**_B. Tab Update:_**
Update the tab structure as per the attached image here
<img width="223" alt="Screenshot 2024-09-12 at 09 51 00" src="https://github.com/user-attachments/assets/2fb8ff0e-a5c2-45db-a2ac-e36fc9dc12d2"> and also remove 'play/pause icon' here 
<img width="169" alt="Screenshot 2024-09-12 at 09 52 46" src="https://github.com/user-attachments/assets/f267b51d-d401-4053-8587-e5a2aaac491d">

**_C. Members Page:_**
- Remove the play/pause icon from the Members page.
- Change terminology from "Members" to "Partners" throughout the site, including the top right tab.
- Reduce the font size of the "Partners" heading.
- Add an image or visual element similar to the homepage.
- Each logo links to a partner page on the C|O website, see below.
- Create individual pages for each partner, including:
- Image of CEO heading up partner company + and short quote (we can ask all new partners for this).
- Company bio
- Partner "News" promo boxes (as per Homepage, but bespoke to their institution) for updates on their AI policy updates, news/reports, and workshops
- Use a layout similar to the events promos on the homepage for partner listings.
- Ensure partner logos match the background (white on white) for a cohesive look, they look cut & pasted.

**Comment by JesseJamesRingham** [comment]
<img width="223" alt="Screenshot 2024-09-12 at 09 51 00" src="https://github.com/user-attachments/assets/2a770f95-07ad-4c80-96a7-a80198d02168">

**Comment by JesseJamesRingham** [comment]
artwork.image examples 
https://vimeo.com/267289442
https://www.aaronkoblin.com/project/flight-patterns/

**Comment by tammyyang** [comment]
Since this is a long list, I believe it's better to do it in phases.
@andyandesign, could you please select the items that can be completed within a working day (8 hours)? We can aim for another update by next Tuesday.

**Reply by aphxbt** [reply]
yea, I’ll try to cover as much as possible each phase.

**Reply by JesseJamesRingham** [reply]
Thank you @andyandesign and @tammyyang, so sorry for the long list, some look like quick edits, but some may take longer. Just think good to get right before proceeding with creative partners. Thank you

**Comment by tammyyang** [comment]
@JesseJamesRingham  if there are any items that you believe must be updated for the launch next Tuesday, please feel free to share them here.

**Comment by aphxbt** [comment]
I’ve updated most of the items, and we can continue to optimize the site in future phases. I left out the partner section for now and will update it once we have all the information ready.

**Reply by tammyyang** [reply]
Agree to leave the partner section for now. You are the best!
